avogadro/qtplugins/cartoons/cartoons.cpp

```diff
@@ -181,107 +163,140 @@ void renderBackbone(const AtomsPairList& backbone, const Molecule& molecule,
 
   CylinderGeometry* cylinders = new CylinderGeometry;
   cylinders->identifier().molecule = &molecule;
-  cylinders->identifier().type = Rendering::BondType;
+  cylinders->identifier().type = Rendering::AtomType;
```

Comment:
It might be worth adding a separate `Rendering::ResidueType` but this is okay for now.

Reply:
Yes, I'm thinking in a good solution for single ray vs cartoon so it works smooth. But I wanted to add at least some selection so the user can work with this.